### DISCUSSION TITLE
Fullscreen playback

### DISCUSSION THREAD

**Discussion by pravdomil** [keiwando/toonsquid#202] [Ideas]
That would be very handy!

**Comment by zaya14** [comment]
I 2nd this! But with more versatility to it. For example you can have multiple playback options along with exporting capabilities. This would be great for people who not only animate who are or planing to make videos on social media platforms like YouTube, etc. What I would love to see for playback is:
1. Zoomed in without HUDs- where you can see each stroke you made on the project with out the huds, tools, being visible
2. Zoomed out without HUDs- where you can see your whole animation/drawing progress all at once
3. Zoomed in with HUDs- same concept but with the huds being visible so people can see the tools you used 
4. Zoomed out with HUDs- same concept but with huds being visible so people can see the tools you used

The idea was from the drawing app called ibis paint I love the different view you could choose from along with exporting. But sadly I don’t think they plan on an animation app, I tried to ask clipstudio but they replied that they would consider it………. That was 2 years ago…… 
I recently found about this app and started experimenting with the hierarchy for rigging and now trying to make a 360 character with replaceable parts using the symbols. This app has soo much potential and looking forward to what can be done!! Hopefully the playback can be added along with the features I mentioned. Thanks for reading!
